swagger/v1/hearing.json

```diff
@@ -7,12 +7,246 @@
+        },
+        "type": {
+          "$ref": "court_application.json#/definitions/type"
+        }
+      }
+    },
+    "cracked_ineffective_trial_relationship": {
+      "type": "object",
+      "properties": {
+        "data": {
+            "$ref": "#/definitions/cracked_ineffective_trial"
+        }
+      }
+    },
+    "cracked_ineffective_trial": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "$ref": "cracked_ineffective_trial.json#/definitions/id"
+        },
+        "type": {
+          "$ref": "cracked_ineffective_trial.json#/definitions/type"
+        }
+      }
+    },
+    "defendant_offence": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "$ref": "offence.json#/definitions/id"
+        },
+        "type": {
+          "$ref": "offence.json#/definitions/type"
+        }
+      }
+    },
+    "defendant_offence_relationship": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "items": {
+            "$ref": "#/definitions/defendant_offence"
+          },
+          "type": "array"
+        }
+      }
+    },
+    "resource_collection": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "items": {
+            "$ref": "#/definitions/resource"
+          },
+          "type": "array"
+        },
+        "included": {
+          "items": {
+            "anyOf":[
+              {
+                "$ref": "hearing_event.json#/definitions/resource"
+              },
+              {
+                "$ref": "provider.json#/definitions/resource"
+              },
+              {
+                "$ref": "court_application.json#/definitions/resource"
+              },
+              {
+                "$ref": "cracked_ineffective_trial.json#/definitions/resource"
+              }
+            ]
+          },
+          "type": "array"
+        }
+      },
+      "required": [
+        "data"
+      ]
+    },
+    "attributes": {
+      "type": "object",
+      "properties": {
+        "court_name": {
+          "$ref": "#/definitions/court_name"
+        },
+        "hearing_type": {
+          "$ref": "#/definitions/hearing_type"
+        },
+        "hearing_days": {
+          "$ref": "#/definitions/hearing_days"
+        },
+        "defendant_names": {
+          "$ref": "#/definitions/defendant_names"
+        },
+        "judge_names": {
+          "$ref": "#/definitions/judge_names"
+        },
+        "prosecution_advocate_names": {
+          "$ref": "#/definitions/prosecution_advocate_names"
+        },
+        "defence_advocate_names": {
+          "$ref": "#/definitions/defence_advocate_names"
+        }
+      }
+    },
     "id": {
       "description": "unique identifier of a hearing",
       "example": "04180ff1-99b0-40b7-9929-ca05bdc767d8",
       "format": "uuid",
       "type": "string"
     },
+    "type": {
+      "description": "Description of the hearing type",
+      "example": "hearings",
```

Comment:
"example": "First hearing", (or plea, or trial, or sentencing - whatever seems more clear)

Reply:
I'd like to keep this example  as "hearings" so it matches the payload - each object comes with id: and type: pairs with the id being a uuid and the type being the name of the object.

Reply:
👍